README.md

```diff
@@ -36,31 +36,53 @@ The Class Request Tool (CRT) is a class reservation system that lets instructors
 3. To add your initial customizations, create a `config/customization.yml` file based on [config/customization.yml.example](config/customization.yml.example).
 4. Create a mailer file `config/mailer.yml.erb` file on [config/mailer.yml.example](config/mailer.yml.example).
 5. Create a `config/database.yml` file based on [config/database.yml.postgres](config/database.yml.postgres). There are no changes required to the database.yml file since it reads the environment varibles in the `.env` file for the database connection.
+6. Read the instructions in [Database Connection](#database-connection) to create a local database or connect to an existing remote database
 
-### Running the app manually
-These instructions are for running the application and database directly on a host.
+### Database Connection
 
-1. Complete all steps in the [Preparation](#preparation) instructions
-2. Run `bundle install`. You will probably have to install OS-vendor supplied libraries to satisfy some gem install requirements.
-3. Create a local database or connect to an existing remote database
+**Database connection options for local development**
+
+#### Option 1: Create a local postgres database
 
-* Option 1: Create a local database
-  * Create a local postgres database with the database name, host, port, and credentials matching the values in `.env` file
+* Create a local postgres database
+  * Option 1A (Recommended) Docker compose database container
+  * Option 1B Create a database on the host machine directly
+* Set the configuration values for the database name, host, port, and credentials in the `.env` file. Read (Configuration)[#configuration] for more information.
+* Run the rake tasks in the ruby environment
   * Run `rake db:schema:load` to automatically load the schema in `./db/schema.rb`.
   * Run `rake db:seed` to seed the database. Make sure you pay attention to the output of this rake task, as it will give you a random password for the Superadmin user created in the database.
+  * Run `rake db:custom_seed` to load placeholder data.
+
+##### Option 1A (Recommended): Docker compose database container
+This option is recommended when running docker compose locally. The local docker compose configuration `docker-compose-local.yml` creates a postgres database instance in a container for the application to connect to locally.
 
-* Option 2: Connect to an existing remote database
-  * Update the database coniguration values in `.env` to connect to a remote database such as the QA database
+A new database will be initialized with the values in the `.env` file if it does not exist already. The postgres hostname in the `.env` configuration should match the name of the container in the docker compose configuration. The data directory in the container is mounted to a directory on the local filesystem.
+
+    ```
+    volumes:
+      - ./postgresql/data:/var/lib/postgresql/data
+    ```
+
+##### Option 1B: Create a database on the host directly
```

Comment:
The 1A/1B thing I think is a little confusing - I'd just flatten this to 1,2,3 since they're really distinct options (docker, local, existing).